source/index.erb

```diff
@@ -10,7 +10,25 @@ title: Energy Globe Award
 
   <section ng-controller="AwardsController">
 
-    <div class="award" ng-repeat="award in awards">
+    <section class="filters">
+      <h2>Kategorien</h2>
+      <ul class="category-filter" ng-repeat="category in categories">
+        <li>
+          <input type="checkbox" id="category_checkbox_{{category}}" ng-model="categoryIsShown[category]">
+          <label for="category_checkbox_{{category}}">{{category}}</label>
+        </li>
+      </ul>
+
+      <h2>Jahre</h2>
+      <ul class="year-filter" ng-repeat="year in years">
+        <li>
+          <input type="checkbox" id="year_checkbox_{{year}}" ng-model="yearIsShown[year]">
+          <label for="year_checkbox_{{year}}">{{year}}</label>
+        </li>
+      <ul>
+    </section>
+
+    <div class="award" ng-repeat="award in awards | categoriesFilter:categoryIsShown | yearsFilter: yearIsShown">
```

Comment:
I don't think I like the idea of using angular filters for the filtering of the awards data.
It's pretty clear that this will get out of hand very quickly -> you need to apply every filter in the html markup which is not very beautiful and I don't like logic like that in the templates